### ISSUE TITLE
An IMOS low cost wave buoy has wandered onto the land - National Wave Archive

### ISSUE THREAD

**Issue by atkinsn** [aodn/content#482]
Just looking at the National Wave Archive for a related purpose, noticed 2 things.

1. There shouldn't be any IMOS buoys yet accessible
2. The IMOS buoy that is visible is currently lost on land

**Comment by kereid** [comment]
perhaps it's attached to this shark? https://www.news.com.au/technology/science/animals/shark-tag-running-around-uni-of-wollongong-campus-researchers-want-it-back/news-story/79e74f6d314286201dbc59d5caf3fa70

**Comment by ana-berger** [comment]
That was me. I tried to publish the file in prod and forgot to delete :facepalm: I did other times before for other files too and deleted them, but forgot this one. Sorry! Should be gone from the portal after 5pm today, if the tile caching is working fine. It is already removed from S3.
Also, about the land position of the buoy, that was a test file that probably had wrong coordinates, so no need to worry about that.
